DrawerKit/DrawerKit/Internal API/PresentationController+Gestures.swift

```diff
@@ -56,3 +56,16 @@ extension PresentationController {
         }
     }
 }
+
+extension PresentationController: UIGestureRecognizerDelegate {
+    func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldReceive touch: UITouch) -> Bool {
+        if let view = gestureRecognizer.view,
+           view.isDescendant(of: presentedViewController.view),
+           let subview = view.hitTest(touch.location(in: view), with: nil),
+           subview is UIControl {
+            return false
+        } else {
+            return true
+        }
```

Comment:
`return (subview is UIControl).isFalse`  ?

Reply:
There's no `isFalse` in DK IIRC.